crispy_forms/layout.py

```diff
@@ -451,6 +453,11 @@ def render(self, form, form_style, context, template_pack=TEMPLATE_PACK, extra_c
             extra_context = {}
         if hasattr(self, "wrapper_class"):
             extra_context["wrapper_class"] = self.wrapper_class
+        # Overwrite form-specified label and field class with field-specific
+        if hasattr(self, "label_class"):
+            extra_context["label_class"] = self.field_class
```

Comment:
I haven't looked into detail but should `self.field_class` be `label_class` here? If not let me know and I'll come back to this properly.